packages/boa/lib/worker.js

```diff
@@ -1,19 +1,35 @@
 'use strict';
 
-const { isMainThread, workerData } = require('worker_threads');
+let isMainThread = false;
+let workerData = null;
+let supportWorkerThreads = false;
+
 const { PythonObject, NODE_PYTHON_HANDLE_NAME } = require('bindings')('boa');
 const { wrap } = require('./proxy');
 
 const TYPE_ID = 'SHARED_PYTHON_OBJECT_TYPE';
+try {
+  require.resolve('worker_threads');
```

Comment:
Could we do the flag though the version of process instead trying?

Reply:
I think a feature-based check is better than a version-based.